src/services/jwt.ts

```diff
@@ -21,5 +18,5 @@ export const createToken = ({ uid, network }) => {
     //
     // see: https://www.theregister.com/2018/04/24/myetherwallet_dns_hijack/
     algorithm: "HS256",
-  })
+  }) as JWTToken
```

Comment:
this is fine but we are using typescript implicit type to discover the return for createToken. Please add it to the createToken signature. 

Also not sure if this should be a service (is not querying external services, db, etc). Maybe would be authentication service but only when we add the new auth (in a different pod) 

@vindard @bodymindarts what do you think?